### ISSUE TITLE
v-bind attribute should be sorted by its real name

### ISSUE THREAD

**Issue by ydfzgyj** [vuejs/eslint-plugin-vue#410]
In new rule `vue/attributes-order`, **v-bind** has an order as same as v-model, but I think it should not be regarded as a single group. For example **v-bind:is** should be placed at first but not together with other attributes.

**Comment by ydfzgyj** [comment]
Does anybody comment on it?

**Comment by michalsnik** [comment]
I commented in PR, I think we can separate those groups and have BINDINGS and TWO_WAY_BINDINGS (although there is only v-model in this case). What do you think @chrisvfritz ?

**Comment by chrisvfritz** [comment]
I actually don't think we need a `BINDINGS` group. The [style guide makes no distinction](https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended) between bound and unbound attributes. In fact, it would seem a little strange to me to group all bound attributes together. For example, `class` and `style` can both have bound and unbound versions on the same element, such as:

```vue
<div
  :class="..."
  class="..."
  :style="..."
  style="..."
/>
```

Also, when changing an attribute between bound and unbound, forcing the attribute to then move would not only slow down development, but also make the diff less readable during code reviews.

As an aside though, in the special cases of `v-on="objectOfListeners` and `v-bind="objectOfAttributes"`, I think those should probably appear at the beginning of the style guide's _Other Attributes_ and _Events_ sections, respectively.

Does that make sense?

**Comment by michalsnik** [comment]
It makes perfect sense indeed @chrisvfritz. I didn't thought about class/style case, and it's soo common.
